File: modul 3/Tugas no 5 praktikum pemrograman 3/prak05.c
#include <stdio.h>

int main() {
    int detik, jam, menit, sisaDetik;
    
    scanf("%d", &detik);

    jam = detik / 3600;
    menit = (detik % 3600) / 60;
    sisaDetik = detik % 60;

    if (jam >= 24) {
        int hari = jam / 24;
        jam %= 24;
        printf("%d hari %02d:%02d:%02d\n", hari, jam, menit, sisaDetik);
    } else {
        printf("%02d:%02d:%02d\n", jam, menit, sisaDetik);
    }

    return 0;
}

File: modul 3/Tugas no 5 praktikum pemrograman 3/prak05.py
def konversi_detik(detik):

  jam = detik // 3600
  menit = (detik % 3600) // 60
  sisa_detik = detik % 60

  if jam >= 24:
    hari = jam // 24
    jam %= 24
    return f"{hari} hari {jam:02d}:{menit:02d}:{sisa_detik:02d}"
  else:
    return f"{jam:02d}:{menit:02d}:{sisa_detik:02d}"

detik = int(input())

hasil = konversi_detik(detik)
print(hasil)